examples/index.html

```diff
@@ -59,6 +59,7 @@ <h1>Demo pages</h1>
 				<pfe-cta pfe-priority="secondary" pfe-variant="wind"><a href="../elements/pfe-health-index/demo">pfe-health-index</a></pfe-cta>
 				<pfe-cta pfe-priority="secondary" pfe-variant="wind"><a href="../elements/pfe-icon/demo">pfe-icon</a></pfe-cta>
 				<pfe-cta pfe-priority="secondary" pfe-variant="wind"><a href="../elements/pfe-icon-panel/demo">pfe-icon-panel</a></pfe-cta>
+				<pfe-cta pfe-priority="secondary" pfe-variant="wind"><a href="../elements/pfe-layouts/demo">pfe-layouts</a></pfe-cta>
```

Comment:
This is an old folder that no longer exists

Reply:
this was auto-added when i compiled, how do I prevent that?

Reply:
@starryeyez024 I removed this